### PR TITLE
[TTAHUB-1856] Update event and session forms with feedback from testing party

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change

- Added event owner to the top of the training report form
- Alphabetized POC dropdown
- Display save draft alert near buttons on complete session page
- Clear "Event must be complete" error if saving draft immediately after
- The complete session page shows as "complete" in the side navigation if returning to a session that was already completed
- Behavior of event status changes was reworked from mockup. Event status is now read-only if the event is "Not started." Once a session is created, the options become "in progress", "suspended", and "complete."
- You can suspend a TR, and "suspended" was added as a tab on the TR landing page
- Noticed that in permissions matrix, only an owner can complete an event

## How to test

Update your dependencies, as @ttahub/common was incremented
Test the behavior described above and the event and sessions forms more generally

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1856

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: src/models/helpers/flowControl.js [hunk 1]
```diff
@@ -6,12 +6,13 @@
  * It can be either a string or an object with properties `name`, `prefix`, and `suffix`.
  * @returns A boolean indicating whether the function name matches the value.
  */
-function isMatch(
-  functionName: string,
-  value: string | { name: string, prefix?: boolean, suffix?: boolean },
-): boolean {
+function isMatch(functionName, value) {
```
Comment (AdamAdHocTeam) [inline src/models/helpers/flowControl.js:9]: Prob can roll this back with the tsx change.

File: src/services/users.js [hunk 2]
```diff
@@ -467,6 +467,9 @@ export async function getTrainingReportUsersByRegion(regionId) {
         },
       },
     ],
+    order: [
+      [sequelize.fn('CONCAT', sequelize.col('User."name"'), sequelize.col('User."email"')), 'ASC'],
```
Comment (AdamAdHocTeam) [inline src/services/users.js:471]: Concat vs two sort array items.

**Review by AdamAdHocTeam — COMMENTED** [review]
Two things to double check:
- I don't have a way to add session to 'not started' events.
- I don't see a tab for 'suspended' on the ar landing.

**Comment by thewatermethod** [comment]
> Two things to double check:

@AdamAdHocTeam Did you install updated dependencies? If you didn't, it might be the issue

**Comment by AdamAdHocTeam** [comment]
> > Two things to double check:
> 
> @AdamAdHocTeam Did you install updated dependencies? If you didn't, it might be the issue

Yes I ran yarn deps. Lets touch base in the AM.

**Review by AdamAdHocTeam — APPROVED** [review]
Overall looks good. 

My issues were related to the common file not properly updating.

**Pull request merged**